content/pt/_index.html

```diff
@@ -43,12 +43,12 @@ <h2>Os desafios da migração de mais de 150 micro serviços para o Kubernetes</
         <br>
         <br>
         <br>
-        <a href="https://events.linuxfoundation.org/events/kubecon-cloudnativecon-europe-2019" button id="desktopKCButton">KubeCon em Barcelona on Maio 20-23, 2019</a>
+        <a href="https://events.linuxfoundation.org/events/kubecon-cloudnativecon-europe-2019" button id="desktopKCButton">KubeCon em Barcelona em Maio 20-23, 2019</a>
```

Comment:
Based on the change below (line 51):
```suggestion
        <a href="https://events.linuxfoundation.org/events/kubecon-cloudnativecon-europe-2019" button id="desktopKCButton">KubeCon em Barcelona em Maio 20-23 de 2019</a>
```